acceptance_test.go

```diff
@@ -14,32 +14,15 @@
 
 package snowflake
 
-import (
-	"context"
-	"database/sql"
-	"encoding/json"
-	"fmt"
-	"os"
-	"strings"
-	"sync/atomic"
-	"testing"
-	"time"
-
-	sdk "github.com/conduitio/conduit-connector-sdk"
-	"github.com/google/uuid"
-	builder "github.com/huandu/go-sqlbuilder"
-	"go.uber.org/goleak"
-
-	"github.com/conduitio-labs/conduit-connector-snowflake/config"
-	"github.com/conduitio-labs/conduit-connector-snowflake/source/iterator"
-)
-
 // driver Configurable Acceptance test driver.
-type driver struct {
-	sdk.ConfigurableAcceptanceTestDriver
+// type driver struct {
+// 	sdk.ConfigurableAcceptanceTestDriver
 
-	idCounter int32
-}
+// 	idCounter int32
+// }
+
+// TODO FIX
```

Comment:
Part of this PR or separately?

Reply:
We are working on tests atm! Just wanted to get core stuff out for review!